core/transactions/attempt_context_impl.cxx

```diff
@@ -764,8 +763,7 @@ attempt_context_impl::query_begin_work(std::optional<std::string> query_context,
     txdata["state"] = tao::json::empty_object;
     txdata["state"]["timeLeftMs"] = overall_.remaining().count() / 1000000;
     txdata["config"] = tao::json::empty_object;
-    txdata["config"]["kvTimeoutMs"] =
-      overall_.config().kv_timeout ? overall_.config().kv_timeout->count() : core::timeout_defaults::key_value_timeout.count();
+    txdata["config"]["kvTimeoutMs"] = core::timeout_defaults::key_value_durable_timeout.count();
```

Comment:
Here should we be fetching the `key_value_durable_timeout` set in the cluster options instead of always setting the default?

Reply:
Do we have access to cluster options somewhere here?

Looking at Java implementation, it seems like they also don't have it on overall config, and rather pull it from global options (which still modifiable, kind of equivalent of our cluster options)

https://github.com/couchbase/couchbase-jvm-clients/blob/7007f9ce455ec88a8266d17a7c117dff6c4dc3a1/core-io/src/main/java/com/couchbase/client/core/transaction/CoreTransactionAttemptContext.java#L274-L275

Reply:
Will do this on a separate PR so this can get merged (since my change doesn't really change the existing behaviour)

Reply:
It was quite simple so I've added it to this PR